angular-sample-spa/README.md

```diff
@@ -10,22 +10,93 @@ The IBM Cloud AppID SDK can be used with React to create a secure single-page ap
 
 ## To run locally
 
-* Clone the repository
+1 Clone the repository.
 ```
-git clone 
+git clone https://github.com/ibm-cloud-security/appid-sample-code-snippets.git
 ```
-* Navigate to application workspace folder.
+2 Navigate to application workspace folder.
 ```
 cd angular-sample-spa
 ```
-* To get the dpendencies for the app installed run
+3 Install dependencies for the application.
 ```
 npm install
 ```
+4 Start the development server. Navigate to http://localhost:4200 to access your application.
+```
+ng serve
+```
 
-* To start the development server run 
+## Detailed instructions on creating your application
+
+
+1 Install the Angular CLI
+```
+npm install -g @angular/cli
+```
+2 Create a workspace and initial application and go to the workspace folder
+```
+ng new angular-sample-spa
+cd angular-sample-spa
+```
+3 Install the IBM Cloud AppID SDK dependency using npm
+```
+npm install ibmcloud-appid-js
+```
+
+4 In the src directory, go to the app folder and import App ID in the `app.component.ts` file, with the following code:
+```
+import AppID from 'ibmcloud-appid-js';
+```
+5 In the AppComponent class initialize the states of the variables to be used in the application
+```
+userName = '';
+errorMessage = '';
+buttonStyle = 'show';
+messageStyle = 'hide';
+errorStyle = 'hide'
+```
+6 Create a loginAction function which when triggered will start the authentication flow and use tokens to get the user's information. 
+```
+async onLoginClick() {
+    const appID = new AppID();
+    try {
+      await appID.init({
+        clientId: '<CLIENT_ID>',
+        discoveryEndpoint: '<WELLKNOWN_ENDPOINT>
+      });
```

Comment:
thoughts on this approach?
```
export class AppComponent implements OnInit{
  appID = new AppID();
  ....
  async ngOnInit() {
    await this.appID.init({
      clientId: '<CLIENT_ID>',
      discoveryEndpoint: '<WELLKNOWN_ENDPOINT>
    });
  }
```
I'm not too familiar with Angular but it looks like we can use [OnInit](https://angular.io/guide/lifecycle-hooks) as part of the lifecycle hooks, which will fire once. Ultimately, we don't want to init AppID every time they click login. Alternatively, you can set a variable called `initialized = false` and do a check in the `onLoginClick` function but will look messier

Reply:
@brmts update readme with latest code

Reply:
It does have the latest code